git-refresher/pull-requests.qmd

```diff
@@ -0,0 +1,99 @@
+
+9. Once you are happy with your title and description, click 'Create pull request' to confirm. 
+
+10. You will then want to check whether there are any merge conflicts. If there are merge conflicts, they will need resolving before you are able to complete your pull request. 
+
+## GitHub Desktop
+
+1. Ensure you are on the development branch. 
+
+2. Click on the 'Branch' menu and 'Create pull request' 
+
+3. Click 'open in browser'. This will open GitHub in your web browser with the pull request form pre-filled. 
+
+4. Add a title and description for you pull request and then click 'Create pull request' 
+
+## VS Code 
+
+## PyCharm 
+::: 
+
+## Reviewing pull requests
+
+When a pull request is created, it's important to review the changes carefully. This process helps ensure that the code meets the project's standards and doesn't introduce any new issues. During the review, you can discuss the changes with the author, suggest improvements, and request additional commits if necessary.
+
+We will practice this by reviewing the pull request you have just made. 
+
+### GitHub Website
+
+1. Go back to the [git-academy-sandbox](https://github.com/dfe-analytical-services/git-academy-sandbox) repository overview page on GitHub, as if you have just come to find the pull request your colleague has asked you to review. 
+
+2. Navigate to the 'Pull requests' tab. 
+
+3. Click on the pull request you want to review, in this case the one you created above. 
+
+![](images/pull-request-in-GitHub.png){fig-alt="Higlighting where open pull requests show in GitHub once the Pull request tab has been opened."}
+
+4. Review the changes by looking at the "Files changed" tab. You can see a side-by-side comparison of the changes made.Add comments by clicking the "+" icon next to the lines of code you want to comment on.
+
+![](images/pull-request-files-changed.png){fig-alt="Higlighting where to view comparison of changes made and make comments on a pull request."}
+
+5. When reviewing pull requests, you may also wish to view changes locally to check the code runs and view any outputs.You can do this by pulling and changing to the branch that is being requested to be merged in.   
+
+6. Once you have reviewed the changes, you can leave general feedback, approve or request changes by clicking the 'review changes' button.
+
+![](images/pull-request-review.png){fig-alt="Higlighting where to provide general feedback, request changes or approve on a pull request."}
+
+7. Once you are happy with the changes, you should approve the pull request. The person who created the request will then get an email letting them know their changes have been approved and can go back and complete the request. 
```

Comment:
Maybe worth adding that it can be worth dropping them a direct message to let them know you've reviewed their PR!